src/esw/brushless.py

```diff
@@ -108,77 +118,91 @@ async def _send_command(self) -> None:
         command = self._command
         self.command_lock.release()
         try:
-            state = await asyncio.wait_for(
-                self.controller.set_position(
-                    position=command.position,
-                    velocity=command.velocity,
-                    velocity_limit=CommandData.VELOCITY_LIMIT,
-                    maximum_torque=command.torque,
-                    watchdog_timeout=MoteusBridge.ROVER_NODE_TO_MOTEUS_WATCHDOG_TIMEOUT_S,
-                    query=True,
-                ),
-                timeout=self.MOTEUS_RESPONSE_TIME_INDICATING_DISCONNECTED_S,
-            )
-            self.moteus_data = MoteusData(
-                position=state.values[moteus.Register.POSITION],
-                velocity=state.values[moteus.Register.VELOCITY],
-                torque=state.values[moteus.Register.TORQUE],
-            )
-            self._check_has_error(state.values[moteus.Register.FAULT])
+            await self._send_command(command)
         except asyncio.TimeoutError:
             if self.moteus_state.state != MoteusState.DISCONNECTED_STATE:
                 rospy.logerr(f"CAN ID {self._can_id} disconnected when trying to send command")
             self._change_state(MoteusState.DISCONNECTED_STATE)
-            return
 
-    def _check_has_error(self, fault_response: int) -> None:
+    async def _send_command(self, command: CommandData) -> None:
         """
-        Checks if the controller has encountered an error. If there is an error, error state is set.
-        :param fault_response: the value read in from the fault register of the moteus CAN message
+        Calls controller.set_position (which controls the moteus over CAN) with the requested command.
+        If the message has not been received within a certain amount of time, disconnected state is entered.
+        Otherwise, error state is entered.
+
+        It is expected that this may throw an error if there is a timeout.
         """
-        has_error = fault_response != 0
-        if has_error:
+        state = await asyncio.wait_for(
+            self.controller.set_position(
+                position=command.position,
+                velocity=command.velocity,
+                velocity_limit=CommandData.VELOCITY_LIMIT,
+                maximum_torque=command.torque,
+                watchdog_timeout=MoteusBridge.ROVER_NODE_TO_MOTEUS_WATCHDOG_TIMEOUT_S,
+                query=True,
+            ),
+            timeout=self.MOTEUS_RESPONSE_TIME_INDICATING_DISCONNECTED_S,
+        )
 
-            self._change_state(MoteusState.ERROR_STATE)
+        moteus_not_found = state is None or not hasattr(state, "values") or moteus.Register.FAULT not in state.values
+        if moteus_not_found:
+            self._handle_error(99)
```

Comment:
make into constant with descriptive name and comment (like the one on line 169)

Reply:
Good call